react-app/src/components/projects/ProjectDetail.tsx

```diff
@@ -246,7 +288,12 @@ const ProjectDetail = (props: IProjectDetail) => {
         <DataCard
           loading={isLoading}
           customFormatter={customFormatter}
-          values={FinancialInformationData}
+          values={Object.fromEntries(
+            Object.entries(FinancialInformationData).map(([k, v]) => [
+              k,
+              formatMoney(v != null ? Number(String(v).replace(/[$,]/g, '')) : 0), //This cast spaghetti sucks but hard to avoid when receiving money as a string from the API.
```

Comment:
How difficult would it be to slap a transform on these columns?

Reply:
Probably wouldn't be too difficult and I think it would go a long way (should be another ticket though). There's not really any reason we can't just keep it as a number consistently, since the API accepts a number when you go to call save/insert/upsert.